chapters/03/1/Expressions.ipynb

```diff
@@ -91,7 +91,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "**Common Operators.** Data science often involves combining numerical values, and the set of operators in a programming language are designed to so that expressions can be used to express any sort of arithmetic. In Python, the following operators are essential.\n",
```

Comment:
I would prefer if this was "...designed so that expressions can be used to express...", i.e., just delete "to".

Reply:
I prefer "programming language are designed so that", not "programming language so that"